bonfire/Modules/Settings/Controllers/GeneralSettingsController.php

```diff
@@ -76,6 +76,30 @@ public function saveGeneral()
         setting('App.dateFormat', $this->request->getPost('dateFormat'));
         setting('App.timeFormat', $this->request->getPost('timeFormat'));
 
+        // Check for an logo to upload
+        if ($file = $this->request->getFile('siteLogo')) {
+
+            if ($file->isValid()) {
+
+                $filename = 'custom.'. $file->getExtension();
+
+                if ($file->move(ROOTPATH .'public/uploads/logo', $filename, true)) {
+
+                           \Config\Services::image()
```

Comment:
```suggestion
                          service('image')
```
I prefer to use the service helper method. That makes it more flexible in case any changes happen upstream.